ketchup/main.py

```diff
@@ -1,121 +1,231 @@
 import datetime
-
+import math
 import dateparser
 import datetime as dt
-from ketchup.luffy import LUFFY
 import ketchup.watch_schedule_builder as wsb
-import ketchup.fun_fact as fun_fact
 
-DDP = dateparser.date.DateDataParser(languages=["en"], settings={"DATE_ORDER": "DMY"})
+# from ketchup.luffy import LUFFY
+# import ketchup.fun_fact as fun_fact
+
+DDP = dateparser.date.DateDataParser(
+    languages=["en"], settings={"DATE_ORDER": "DMY"})
 
 
 def get_date(
-    prompt: str, require_future: bool = False, require_past: bool = False
+        prompt: str, require_future: bool = False, require_past: bool = False
 ) -> datetime.datetime:
     while True:
         try:
             user_date = input(prompt)
             user_date = DDP.get_date_data(user_date).date_obj
         except ValueError:
-            print("Sorry, please only enter a date. an example would be '10 Aug 2023' ")
+            print("Sorry, please only enter a date. "
+                  "An example would be '10 Aug 2023' ")
             continue
         if require_future:
             if user_date.date() < dt.date.today():
-                print("Sorry, please only enter a future date")
+                print("Sorry, please only enter a future date.")
                 continue
         if require_past:
             if user_date.date() > dt.date.today():
-                print("Sorry, please only enter a past date")
+                print("Sorry, please only enter a past date.")
                 continue
         else:
             break
     return user_date
 
 
+def calculate_episodes_from_hours(episodes, episode_duration_mins=24):
```

Comment:
move this to utils.py

Reply:
allow user to set episode duration if they want.... default to the 24 minutes, notify the user of that choice, but allow them to change this.

Reply:
rename this function maybe to something like "calculate watch length in time"?  or something

Reply:
Moved to a new file called utils.py and renamed it.

Additionally if a user chooses option "1"  when they run the ketchup.py file, they can set the episode duration of their choice.